abstract_specification.md

```diff
@@ -616,17 +621,17 @@ create(string uri, Arrow.Schema schema, string[] index_column_names, platform_co
 
 Parameters:
 
-- uri - location at which to create the object
-- schema - an Arrow Schema defining the per-column schema.
-- index_column_names - an list of column names to use as index columns, aka "dimensions" (e.g., `['cell_type', 'tissue_type']`). All named columns must exist in the schema, and at least one index column name is required. Index column order is significant and may affect other operations (e.g. read result order). The `soma_joinid` column may be indexed.
-- [platform_config](#platform-specific-configuration) - optional storage-engine specific configuration
-- [context](#long-lived-context-data) - optional context to use for this new object.
+- `uri`: location at which to create the object.
+- `schema`: an Arrow Schema defining the per-column schema.
+- `index_column_names`: a list of column names to use as index columns, also known as "dimensions" (e.g., `['cell_type', 'tissue_type']`). All named columns must exist in the schema, and at least one index column name is required. Index column order is significant and may affect other operations (e.g., read result order). The `soma_joinid` column may be indexed.
+- [`platform_config`](#platform-specific-configuration): optional storage-engine specific configuration.
```

Comment:
we should also replace all instances of `storage-engine specific` with `storage-engine–specific` (or `storage engine–specific`)

Reply:
thanks @thetorpedodog -- i had indeed missed more of those than i realized -- appreciate it!